### ISSUE TITLE
New Account VoiceOver: add image button label is not accessible

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-iOS#11489]
Steps to reproduce:

1. Log out of the WPiOS app.
1. Go to Settings > General > Accessibility > VoiceOver and toggle the setting on.
1. Go to WPiOS and double tap the "Sign up for WordPress.com" button.
1. Follow the prompts until you reach the New Account screen.
1. Tap the "Add Image" button to find that doesn't have a label.

Result: only "button" is announced when you tap on the "Add Image" button on the New Account screen. ([5s](https://cloudup.com/cV5xWztv8Rx))

<img src="https://user-images.githubusercontent.com/1119271/56249741-1c5a7180-606a-11e9-8431-5d1012e6616f.png" width="270" alt="voiceover-add-image-button-is-not-announced">
<p style="clear:both;"><sup>Tested with WP Internal 12.2.0.20190412 on iPhone 6S iOS 12.2.</sup></p>

<p style="clear:both;"><sub>(internal reference: p77Llu-bQ6-p2 h/t @etoledom)</sub></p>
